datasets/me/acp_peps/crawler.py

```diff
@@ -0,0 +1,162 @@
+import io
+from csv import DictReader
+from datetime import datetime
+from typing import List, Dict, Optional, Tuple
+from zipfile import ZipFile, BadZipFile
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+from zavod.shed.trans import (
+    apply_translit_full_name,
+    make_position_translation_prompt,
+)
+
+TRANSLIT_OUTPUT = {
+    "eng": ("Latin", "English"),
+}
+POSITION_PROMPT = prompt = make_position_translation_prompt("cnr")
+
+
+def extract_latest_filing(
+    dates: List[Dict[str, object]]
+) -> Tuple[Optional[str], Optional[int]]:
+    if not dates:
+        return None, None
+
+    # Sort the dates in descending order
+    dates_sorted = sorted(
+        dates,
+        key=lambda item: datetime.strptime(item["datum"], "%Y-%m-%d"),
```

Comment:
if they're already in this format, they're already sortable and there's no need to parse them, right? That could simplify this code a touch.